README.md

```diff
@@ -39,21 +39,27 @@ Immediate needs for this repo are:
 
 Make sure you have Node.js 8.x [installed](https://nodejs.org/) and then follow these steps to get up and running:
 ```shell
-# 1) create a new evergreen app
-$ npx create-evergreen-app my-app
+# 1) Install a create-evergreen-app globally
+$ npm install -g create-evergreen-app
```

Comment:
I think `npx` greatly simplifies the installation steps and I don't think CEA really needs to be installed globally / maintained at all times by the developer.  

By using `npx`, it also means the developer would always get the latest and greatest version of the package when starting a new project.  (which would probably be the "API" needed to solve something #32 )

I personally tend to avoid globally installed packages myself in all projects, except for npm and Yarn and I think `npx` is a nice compromise around this.

Reply:
We can always make a distinction between "quick start" and advanced steps when we start adding more options to the CLI, in which we would probably have some basic API docs.

Reply:
Understandable. I'll change it back and just update it to include cd my-app step.